packages/build/releaseNotes/build.md

```diff
@@ -1,5 +1,13 @@
 # @labkey/build
 
+### version TBD
+*Released*: 13 March 2024
+- Issue 49870: add passwordGuage.js dependency to app.template.xml
```

Comment:
Ideally, the `PasswordGauge` would be a React component that we can then expose for usage in LKS. That said, I think this is a fine trade-off for now.